project.clj

```diff
@@ -1,19 +1,18 @@
-(def schema-tools-version "0.9.0")
-(def schema-version "1.1.3")
+(def schema-tools-version "0.9.1")
+(def schema-version "1.1.7")
 
-(defproject threatgrid/flanders "0.1.14-SNAPSHOT"
+(defproject threatgrid/flanders "0.1.14"
```

Comment:
You should keep the `-SNAPSHOT` prefix in the PR. The release is always made from master branch in this way:

1. Remove `-SNAPSHOT` suffix in `project.clj`
2. Commit the change `git commit -m “Release ${version}”`
3. Tag the version `git tag -m “$version” “$version”`
4. Deploy to clojars
5. Bump the version with the `-SNAPSHOT` prefix in project.clj and commit `git commit -m “Bump to ${newversion}”`
6. Push everything to upstream `git push` and `git push --tag`

Reply:
Modified to include -SNAPSHOT again